lib/screens/dungeon/dungeon_info_subtab.dart

```diff
@@ -204,6 +205,8 @@ class ExpCoinTable extends StatelessWidget {
   final SubDungeon sd;
   const ExpCoinTable(this.sd, {Key key}) : super(key: key);
 
+  final _format = NumberFormat("###,###,###,###");
```

Comment:
Pretty sure you can replace this with NumberFormat.decimalPattern()

Reply:
Ok, I saw that function and wasn't sure exactly what it did since all the docs mention is a hardcoded constant that is itself undocumented.

Reply:
Yeah those docs could be better